### PR TITLE
[196] Handle ActionDefinition, ActionUsage and SuccessionAsUsage

### PR DESCRIPTION
**Pull request by adaussy**
Bug: https://github.com/eclipse-syson/syson/issues/196

### CODE REVIEW

File: backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/SysMLElementSerializer.java [hunk 1]
```diff
@@ -127,15 +133,22 @@ public class SysMLElementSerializer extends SysmlSwitch<String> {
...
         super();
         this.lineSeparator = lineSeparator;
         this.indentation = indentation;
         this.nameDeresolver = nameDeresolver;
+        if (reportConsumer == null) {
+            this.reportConsumer = r -> {
+            };
+        } else {
+            this.reportConsumer = reportConsumer;
+
```
Comment (AxelRICHARD) [inline backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/SysMLElementSerializer.java:146]: please do not let empty lines like this one, here and everywhere else

File: backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/SysMLElementSerializer.java [hunk 2]
```diff
@@ -208,6 +226,108 @@ public String casePartUsage(PartUsage partUsage) {
         return this.appendDefaultUsage(this.newAppender(), partUsage).toString();
     }
 
+    @Override
+    public String caseActionUsage(ActionUsage actionUsage) {
+
```
Comment (AxelRICHARD) [inline backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/SysMLElementSerializer.java:231]: Please do not let empty lines on first lines of methods, here and everywhere else.

File: backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/utils/NameDeresolver.java [hunk 3]
```diff
@@ -78,7 +79,9 @@ public String getDeresolvedName(Element element, Element context) {
         if (deresolvingNamespace == null) {
             qualifiedName = getQualifiedName(element);
         } else {
+            // A element is either reachable form its containment tree or via a reference Membership#memberElement
```
Comment (AxelRICHARD) [inline backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/utils/NameDeresolver.java:82]: A element -> An element

File: backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/utils/SysMLKeywordSwitch.java [hunk 4]
```diff
@@ -71,6 +75,17 @@ public String caseAttributeUsage(AttributeUsage object) {
...
+    public String caseActionDefinition(ActionDefinition object) {
+        return ACTION;
+    }
+
+    @Override
+    public String caseActionUsage(ActionUsage object) {
+        return ACTION;
+    }
+
+
```
Comment (AxelRICHARD) [inline backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/utils/SysMLKeywordSwitch.java:88]: Please do not let empty lines like this, here and everywhere else.

File: backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/FeatureImpl.java [hunk 5]
```diff
@@ -260,14 +260,12 @@ protected EClass eStaticClass() {
...
-     * <!-- begin-user-doc --> <!-- end-user-doc -->
-     *
-     * @generated NOT
-     */
     @Override
     public EList<Feature> getChainingFeature() {
-        List<Feature> data = new ArrayList<>();
+        List<Feature> data = this.getOwnedFeatureChaining().stream()
+                .map(FeatureChaining::getChainingFeature)
+                .filter(f -> f != null)
```
Comment (AxelRICHARD) [inline backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/FeatureImpl.java:267]: `.filter(f -> f != null)` -> `.filter(Objects::nonNull)`

File: backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/SysMLElementSerializer.java (hunk 1 above)
Comment (adaussy) [reply backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/SysMLElementSerializer.java:146]: Done

File: backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/SysMLElementSerializer.java (hunk 2 above)
Comment (adaussy) [reply backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/SysMLElementSerializer.java:231]: Done

File: backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/utils/NameDeresolver.java (hunk 3 above)
Comment (adaussy) [reply backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/utils/NameDeresolver.java:82]: Done

File: backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/FeatureImpl.java (hunk 5 above)
Comment (adaussy) [reply backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/FeatureImpl.java:267]: Done

**Review by AxelRICHARD — APPROVED** [review]

**Pull request merged**